Appraisals

```diff
@@ -18,7 +18,14 @@ appraise 'rails-5' do
 end
 
 appraise 'rails-5-1' do
-  gem 'rails', '~> 5.1'
+  gem 'rails', '~> 5.1.0'
+  gem 'sqlite3'
```

Comment:
why does it need the news gems?

Reply:
Well I made sure that this appraisal is only for `5.1.x` and not for `5.x`. The `sqlite` I'm not sure, to be honest, but it's in all of the other appraisal groups as well. I think the tests were failing without it.